### DISCUSSION TITLE
Can <dialog> be used in WinBox.JS?

### DISCUSSION THREAD

**Discussion by jfbaro** [nextapps-de/winbox#159] [Ideas]
Can <dialog> be used in WinBox.JS?

**Comment by ts-thomas** [comment]
You can use `<dialog>` like every element inside the Winbox body content. But you will get an inserted dialog box which might not fits your desired behavior. Winbox has its own `modal` state for this purpose. You can configure a Winbox instance as a standard confirm, alert or prompt box (which could also be non-modal).
